controllers/contacts.js

```diff
@@ -0,0 +1,79 @@
+const fs = require("fs").promises;
+
+const path = require("path");
+const contactsPath = path.resolve("models/contacts.json");
+
+const listContacts = async () => {
+  const data = await fs.readFile(contactsPath, "utf-8");
+  return JSON.parse(data);
+};
+
+const getContactById = async (id) => {
+  const data = await fs.readFile(contactsPath);
+  const parseData = JSON.parse(data);
```

Comment:
nie ma potrzeby za każdym razem definiowania dwóch zmiennych aby otrzymać zparsowane dane z pliku 

używając poniżego kodu osiągniemy ten sam efekt

```suggestion
  const data = await fs.readFile(contactsPath, 'utf-8');
```